SU-SWS/vpue_undergrad_subtheme
# UE-263: Style and set up a hard-coded fat footer with links

# READY FOR REVIEW

# Summary
VPUE fat footer below the local footer and ABOVE global footer. 
Tightening up the webpack output
Removing commented out old code.

# Review By (Date)
- Soon

# Urgency
- High

# Steps to Test

1. Pull in the changes.
2. run `npm run publish && lando drush cr`
3. Verify build works and the footer looks as wanted.

# Affected Projects or Products
- VPUE Subtheme
- Footer

# Associated Issues and/or People
- [UE-263](https://stanfordits.atlassian.net/browse/UE-263): Style and set up a hard-coded fat footer with links
- @jdwjdwjdw 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
